components/bl-carousel-component/styles/index.less

```diff
@@ -0,0 +1,172 @@
+// you can use LESS variables from the current Theme and app extensions of the UI-Container
+
+.bl-customComponent-carousel {
+  position: relative;
+
+  .carousel {
+    &__list {
+      width: 100%;
+      overflow: hidden;
+    }
+
+    &__item {
+      display: none;
+      float: left;
+      width: 100%;
+      margin-right: -100%;
```

Comment:
Did you get styles from the reference library? Because I can't find float or negative margins at the source.

Reply:
This style I get from devtools
![image](https://user-images.githubusercontent.com/99363993/185609875-640b2cde-2ae3-468d-900b-64a571439154.png)